demo/talkingdata-adtracking-fraud-detection/predict_server.py

```diff
@@ -165,7 +167,7 @@ def make_app():
     global global_args
```

Comment:
global at module level.
this global stmt can be just removed

Reply:
done.